site/content/docs/single-sign-on/saml.md

```diff
@@ -41,10 +39,35 @@ Please make sure that Checkly is sent the email address back from your SSO Idp.
 
 ## Testing the integration
 
-Once that is taken care of, you should be able to log in to Checkly via SSO already. Entering an email address associated with the domain you have provided in the login prompt should result in the password field disappearing:
+Once that is taken care of, logging in to Checkly via SSO is ready. Entering an email address associated with the domain you have provided in the login prompt should result in the password field disappearing:
 
 ![checkly login prompt without password screenshot](/docs/images/single-sign-on/checkly-login-prompt-sso.png)
 
 After submitting the Checkly login form, you should be redirected to your SSO login interface. Completing the login procedure will then lead you to your existing Checkly account, if you have one, or to the new account creation screen, in case you don't.
 
-> Note: once SAML SSO has been set up, you will still need to invite new users from your organization to your Checkly account, as they will not be added automatically.
\ No newline at end of file
+> Note: once SAML SSO has been set up, you will still need to invite new users from your organisation to your Checkly account, as they will not be added automatically.
```

Comment:
They don't need to be "invited", per se. They just need to be assigned access to the "Checkly App" in their SSO dashboard. 

i.e. in Okta you have a list of all the apps your users can possibly use / that have been setup with your company. Then the admin has to "assign access" to the apps to each user (or group or whatever).

At that point, once they have access via their IdP, they can just login to Checkly and be added automatically to their respective Checkly Account ("organisation").

Reply:
Do you think I should remove the note completely? 
So if a customer sets up SSO, they don't have to invite users from Checkly? What happens if they do?

Reply:
Correct, they do not need to invite them. If they do, the user will still be added to their companies account with whichever method they used to signin (most likely their SSO IdP in this scenario, right). 

I think you can probably remove this note entirely actually. The general flow of setup your SSO connection with checkly and your users can signin and be auto provisioned is explained elsewhere thoroughly enough on this page, right. 

What do you think?

Reply:
Agreed :D